src/Test/TestDamageStrategy.java

```diff
@@ -24,6 +24,7 @@
 import Units.UnitModel;
 import Units.UnitModel.UnitEnum;
 import Units.Weapons.WeaponModel;
+import Units.Weapons.Damage.ChassisDamage;
```

Comment:
An extra unneeded import, I see :v

Reply:
Aw, snap. I added that to play with it locally, and added it to the commit on accident. On the other hand, perhaps it *should* be included. Maybe `TestDamageStrategy` should compare each type to `StandardDamage` or something, and give a warning if values are too far diverged. Might help with ensuring a new system is at least vaguely sane. What you think?